ods_ci/tests/Resources/OCP.resource

```diff
@@ -102,7 +102,15 @@ Wait For Pods To Be Ready
     ...    status_only=${FALSE}
     ${rc}    ${out}=    Run And Return Rc And Output
     ...    oc wait --for=condition=ready pod -l ${label_selector} -n ${namespace} --timeout=${timeout}
-    Should Be Equal As Numbers    ${rc}    ${0}
+    ${pass}=    Run Keyword And Return Status    Should Be Equal As Numbers    ${rc}    ${0}
+    IF    ${pass} == ${FALSE}
+        ${rc}    ${events}=    Run And Return Rc And Output    oc get events -n ${namespace}
+        Log    ${events}
+        ${rc}    ${podlogs}=    Run And Return Rc And Output
+        ...    oc logs -f -l ${label_selector} --all-containers -n ${namespace}
+        Log    ${podlogs}
+        Fail
```

Comment:
I think you can add some message using msg parameter to make the failure more clear and readable in console. Something like "Pod is not up and running in give time" or something on the similar line

Reply:
I thought it was redundant, the keyword is called `Wait for pod to be ready` if it fails you see it soon in the logs and the pod is not ready. But I can add the msg if you think it's beneficial

Reply:
it was chip, so I did it.

Reply:
hmm

Reply:
?